### PR TITLE
Externalize and style the footer

### PR DESCRIPTION
**Pull request by mjoyce91**
Moved footer to its own component and added non-pixel-perfect style from the wireframe. Makes use of snapshot testing.

### CODE REVIEW

File: src/Components/Footer/Footer.test.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+describe('DetailsComponent', () => {
+  let footer = null;
+
+  it('is defined', () => {
+    footer = shallow(<Footer />);
+    expect(footer).toBeDefined();
+  });
+
+  it('takes props', () => {
```
Comment (jseppi) [inline src/Components/Footer/Footer.test.jsx]: Name of test should be something like "matches snapshot"

File: src/Components/Footer/Footer.test.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJSON from 'enzyme-to-json';
+import Footer from './Footer';
+
+describe('DetailsComponent', () => {
```
Comment (jseppi) [inline src/Components/Footer/Footer.test.jsx]: Should be FooterComponent

File: src/Components/Footer/Footer.test.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  let footer = null;
```
Comment (jseppi) [inline src/Components/Footer/Footer.test.jsx]: No reason to instantiate the variable here. Can just do it in 'it' cases

**Comment by mjoyce91** [comment]
@burgwyn @jseppi - thanks for feedback. See updates.

**Pull request merged**